tests/test_header.py

```diff
@@ -33,3 +43,37 @@ def is_linked_fixture(self, request):
         sectPr_cxml, expected_value = request.param
         header = Header(element(sectPr_cxml), None, WD_HEADER_FOOTER.PRIMARY)
         return header, expected_value
+
+    @pytest.fixture
+    def body_fixture(self, part_prop_, block_item_):
+        # this is what i want to do but it throws an error...
+        # > sectPr = element('w:sectPr/w:headerReference{w:type=default,r:id=rId8}')
+        # < lxml.etree.XMLSyntaxError: Namespace prefix r for id on headerReference is not defined, line 2, column 50
+        # how do i fix?
```

Comment:
I think this should go away once you have the `CT_HeaderThingaMaJig` element defined. Getting the error from lxml.etree... always seems to indicate an element not defined/registered yet.

If it doesn't fix it, let me know, it could be the cxml parser needs a tweak to recognize attribute namespace prefixes.